deploy/ansible/worker/inventory.yml

```diff
@@ -0,0 +1,49 @@
+all:
+  children:
+    nucypher:
+      children:
+        cassandra:
+          children: # add or remove children as needed
+
+            #### digital ocean uses default user "root" so we have to group these separately
+            digitalocean: # this can be anything... "my_nodes" or "home"
+              vars:
+                default_user: "root"
+              hosts:
+                192.168.1.1:
+                  WORKER_ACCT_KEYSTORE_PATH: "/Users/macperson/Library/Ethereum/goerli/keystore/UTC--2020-01-21T02-15-18.405738000Z--02e8cbf55e781ad4ca331fe5274be93814d760d0"
+                  NUCYPHER_WORKER_ADDRESS: "0x02e8cbf55E781AD4cA331fe5274Be93814D760D0"
+                  NUCYPHER_STAKER_ADDRESS: "0xD9b6B55b005f1B23b45a9a4aC9669deFac6dAd67"
+
+            #### amazon configures new instances with a .pem keypair based auth
+            # so we need this ansible_ssh_private_key_file variable
+            amazon:
+              vars:
+                ansible_ssh_private_key_file: ~/Downloads/damon-ansible-testing.pem
+                default_user: "ubuntu"  # or root for digital ocean
+              hosts:
+                # add a host for each worker/staker
+                cassandra1.mystakednodez.com:
+                  WORKER_ACCT_KEYSTORE_PATH: "/home/ubuntu/.ethereum/goerli/keystore/UTC--2020-01-21T02-15-33.342507000Z--d9e7eC6f9bB558c739CDdbAD5c38F170F1571077"
+                  NUCYPHER_WORKER_ADDRESS: "0xd9e7eC6f9bB558c739CDdbAD5c38F170F1571077"
+                  NUCYPHER_STAKER_ADDRESS: "0x4ffaEAe86c6A6f2C810F048877fbe1FBB2B27606"
+                86.75.30.9:
+                  WORKER_ACCT_KEYSTORE_PATH: "/home/ubuntu/.ethereum/goerli/keystore/UTC--2020-01-21T02-15-33.342507000Z--d9e7eC6fddde58c739CDdbAD5c38F170F1571077"
+                  NUCYPHER_WORKER_ADDRESS: "0xd9e7eC6fddde58c739CDdbAD5c38F170F1571077"
+                  NUCYPHER_STAKER_ADDRESS: "0x4ffaEAe8aaee6f2C810F048877fbe1FBB2B27606"
+
+          # these variables apply to everything under 'cassandra'
+          vars:
+            network_name: cassandra
+            geth_options: "--goerli"
+            geth_dir: '/home/nucypher/geth/.ethereum/goerli/'
+            geth_container_datadir: "/root/.ethereum/goerli"
+            etherscan_domain: goerli.etherscan.io
+
+            # these can be overridden at the instance level if desired
+            NUCYPHER_KEYRING_PASSWORD: xxxxxxxxxxxxxxxxxxxxxxxpanda
+            NUCYPHER_WORKER_ETH_PASSWORD: yyyyyyyyyyyyyyyyyyyystainpants
```

Comment:
Beyond the scope of  *this* PR - but note that it is possible to use inline encrypted ansible vault strings here, if the user is familiar with ansible tooling.